### ISSUE TITLE
Auto-Download new chapters does not work when settings are set to "Exclude one category" and nothing else

### ISSUE THREAD

**Issue by the-codinator** [tachiyomiorg/tachiyomi#6541]
### Steps to reproduce

1. Refresh Library (auto or manual)
2. See new chapters detected (Side Note: 1 manga refresh failed with 404 since the site has taken down that manga - not sure if this could be related to the cause.)
3. See new chapters listed in the "Updates" tab
4. Notice that these chapters aren't getting automatically added to the download queue anymore.

Additional info: This is happening genetically for all manga. The manga is not excluded by category in the "Download new chapters" section of "Download" settings. Also tried this with CBZ Archive mode Enabled as well as Disabled, but that does not seem to affect the issue.

### Expected behavior

Add new chapters to download queue and start download

### Actual behavior

New chapters are not added to download queue.
Currently having to manually download each ever since version update.

### Tachiyomi version

working on 0.12, stopped working after 0.13.0 update and still not working on 0.13.1

### Android version

Android 9

### Device

OnePlus 3

### Other details

Bunch of screenshots regarding issue and my configuration options.
![Screenshot_20220202-093850](https://user-images.githubusercontent.com/4411711/152091932-80e0ee41-3450-4a9b-8b5c-3698ae57fb29.jpg)
![Screenshot_20220202-093901](https://user-images.githubusercontent.com/4411711/152091951-a5741dc9-62e8-44cd-949c-44251d249645.jpg)
![Screenshot_20220202-093931](https://user-images.githubusercontent.com/4411711/152091995-297c20cc-f535-4c7a-baf3-4eba6660408a.jpg)
![Screenshot_20220202-093924](https://user-images.githubusercontent.com/4411711/152092011-55717845-10e7-4e11-b1e1-69a123bfa637.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
I see you haven't included any categories in global updates in your last screenshot. Is that intentional?

**Comment by the-codinator** [comment]
@arkon I basically didn't select anything to explicitly include, which should default to "include all" (and excluded only 1 category). The global update _does_ seem to work, coz when I hit the "refresh" button on the top right of the library tab, then I do see the update notification showing progress for my 41 manga. I also see new chapters in the "Updates" tab after the refresh which I then need to manually click on their download buttons...

**Comment by the-codinator** [comment]
@arkon I seem to have found an issue... in the Download categories, if I don't explicitly include/exclude anything, then auto-download works. It says
```Include: All
Exclude: None```

But when I exclude just one category and leave the rest as unselected, then NOTHING gets auto-download. And the setting shows as
```Include: All
Exclude: idk```

This scenario is what is shown above in the screenshot

Something definitely seems to be wrong here...

Also, the way the Global Updates setting works is, if I only exclude 1 category and don't select anything else then it shows
```Include: None
Exclude: fin```
and this works fine...

So it seems to me that the notion of "All" and "None" seems to be getting applied incorrectly here...

I would recommend someone else also to try this out since it seems like a pretty major settings configuration bug in my opinion.

For now, what I have done to work around the issue is to select nothing (no include / no exclude) in the auto-download category settings.

**Comment by Mifuto** [comment]
Can confirm auto-download doesn't work if one category is excluded

**Comment by FourTOne5** [comment]
Didn't notice it earlier but this is closed by #6635

**Comment by chacho2me** [comment]
please do add NINEMANGA as well. 

webview shows updated chapters but not on the app

**Comment by the-codinator** [comment]
This issue is not resolved - Opened #7013 as follow-up
